src/lssdp.c

```diff
@@ -303,7 +303,7 @@ int lssdp_socket_read(lssdp_ctx * lssdp) {
     socklen_t address_len = sizeof(struct sockaddr_in);
     lssdp_packet packet = {0};
 
-    ssize_t recv_len = recvfrom(lssdp->sock, buffer, sizeof(buffer), 0, (struct sockaddr *)&address, &address_len);
```

Comment:
![image](https://user-images.githubusercontent.com/25633294/134892882-e4106014-62c5-4768-8368-0a50fdf83677.png)
This is not a spelling mistake, it should be the header file that is under-protected